test/views/declaration/PreviousDocumentsViewSpec.scala

```diff
@@ -230,14 +230,14 @@ class PreviousDocumentsViewSpec extends UnitViewSpec with ExportsTestData with S
       view.select("form>table>thead>tr>th:nth-child(2)").text() must be("supplementary.previousDocuments.documentType.label")
       view.select("form>table>thead>tr>th:nth-child(3)").text() must be("supplementary.previousDocuments.documentReference.label")
       view.select("form>table>thead>tr>th:nth-child(4)").text() must be("supplementary.previousDocuments.goodsItemIdentifier.label")
-      view.select("form>table>thead>tr>th:nth-child(5)").text() must be("supplementary.packageInformation.remove")
+      view.select("form>table>thead>tr>th:nth-child(5)").text() must be("supplementary.previousDocuments.remove")
```

Comment:
Do we have a better CSS selector for this, or not yet?

Reply:
Don't know.  Was just fixing a typo.